chainimport/headers_import.go

```diff
@@ -203,9 +206,35 @@ func (h *headersImport) Import() (*ImportResult, error) {
 	result.StartHeight = regions.importStartHeight
 	result.EndHeight = regions.importEndHeight
 
+	// TODO(mohamedawnallah): process the overlap region. This mainly
```

Comment:
We don't need to handle these atm, as we only import headers if the source is blank anyway, right?